micropython/build.sh

```diff
@@ -21,24 +24,55 @@ mkdir -p "$PREFIX_UPYTH_BUILD" "$PREFIX_UPYTH_MARKERS"
 
 
 #
-# Apply patches
+# Apply patches and copy files
 #
 # FIXME: prefix patch files with numbers to ensure correct order
-for patchfile in "$PREFIX_UPYTH_PATCHSET"/*.patch; do
+for patchfile in "$PREFIX_UPYTH_CONFIG"/patches/*.patch; do
 	if [ ! -f "$PREFIX_UPYTH_MARKERS/$(basename "$patchfile").applied" ]; then
 		echo "applying patch: $patchfile"
 		patch -d "$PREFIX_UPYTH_SRC" -p1 < "$patchfile"   # FIXME: should be -p1
 		touch "$PREFIX_UPYTH_MARKERS/$(basename "$patchfile").applied"
 	fi
 done
+cp -a "${PREFIX_UPYTH_CONFIG}/files/001_mpconfigport.mk" "${PREFIX_UPYTH_SRC}/ports/unix/mpconfigport.mk" && echo "Copied mpconfigport.mk!"
+
+
+#
+# Micropython internal use stack/heap size (not actual application stack/heap)
+# May be overwritten in _targets/build.project.*
+#
+if [[ ! $UPYTH_STACKSZ ]]; then
+	UPYTH_STACKSZ="4096"
+fi
+if [[ ! $UPYTH_HEAPSZ ]]; then
+	UPYTH_HEAPSZ="16384"
+fi
+
+
+#
+# Architecture specific flags/values set
+#
+if [ "${TARGET_FAMILY}" = "armv7m7" ]
+then
+	export LDFLAGS_EXTRA=" -Wl,-q ${CFLAGS}"
+	export STRIPFLAGS_EXTRA="--strip-unneeded"
+elif [ "${TARGET_FAMILY}" = "ia32" ]
+then
+	export STRIPFLAGS_EXTRA="--strip-all"
+else
+	echo "Error: there is no micropython port for chosen architecture"
```

Comment:
I would issue a warning, not a direct build breakage, I've tested the `micropython` to work on `imx6ull` target without many issues (only basic manual tests) - just setting LDFLAGS_EXTRA to CFLAGS + `-Wl,-z,max-page-size=0x1000`

Maybe we can use one of the `b_*` functions to output the log in color?

Reply:
From what I see we export `b_log`, `b_die` and `b_install`. The warning is hardly visible if `b_log` is used. 

The best option would be to use `b_die` which halts the build and prints message in red.